src/actions/actions.js

```diff
@@ -43,10 +43,13 @@ export function importNotebook(newState) {
 export function importFromURL(importedState) {
   return (dispatch) => {
     dispatch(importNotebook(importedState))
+    dispatch(updateAppMessages('Notebook successfully imported from URL.'))
     return Promise.resolve()
   }
 }
 
+//  dispatch(updateAppMessages(`gonna run all cells. ${(new Date()).toString()}`))
```

Comment:
can cut this comment

Reply:
do you wish to retain this comment?

Reply:
meant to cut this before, apologies / thanks for catching again